pctasks/run/Dockerfile

```diff
@@ -1,8 +1,5 @@
 FROM python:3.9-slim
 
-RUN apt-get update && \
-    apt-get install -y build-essential git
```

Comment:
I think we don't need a C compiler, since we aren't installing rasterio from source.